components/ProductImage/style.js

```diff
@@ -6,7 +6,6 @@ const placeholderOffset = (1.0 - placeholderIconScale) * 50;
 
 const container = css({
   position: 'relative',
-  borderRadius: 3,
```

Comment:
why did you remove this?

Reply:
<img width="505" alt="screen shot 2018-05-15 at 14 27 40" src="https://user-images.githubusercontent.com/292008/41404912-bd269938-6fc8-11e8-9982-52ad43caf502.png">